static/js/containers/admin/EditChannelContributorsPage_test.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+import { assert } from "chai"
+
+import { makeChannel, makeContributors } from "../../factories/channels"
+import { actions } from "../../actions"
+import { SET_CHANNEL_DATA } from "../../actions/channel"
+import { formatTitle } from "../../lib/title"
+import { editChannelContributorsURL } from "../../lib/url"
+import IntegrationTestHelper from "../../util/integration_test_helper"
+
+describe("EditChannelContributorsPage", () => {
+  let helper, renderComponent, channel, contributors
+
+  beforeEach(() => {
+    channel = makeChannel()
+    contributors = makeContributors()
+    helper = new IntegrationTestHelper()
+    helper.getChannelStub.returns(Promise.resolve(channel))
+    helper.getChannelsStub.returns(Promise.resolve([channel]))
+    helper.getChannelContributorsStub.returns(Promise.resolve(contributors))
+    helper.getPostsForChannelStub.returns(
+      Promise.resolve({
+        pagination: {},
+        posts:      []
+      })
+    )
+    helper.getFrontpageStub.returns(
+      Promise.resolve({
+        pagination: {},
+        posts:      []
+      })
+    )
+    helper.updateChannelStub.returns(Promise.resolve(channel))
+    helper.getProfileStub.returns(Promise.resolve(""))
+    renderComponent = helper.renderComponent.bind(helper)
+    window.scrollTo = helper.sandbox.stub()
+  })
+
+  afterEach(() => {
+    helper.cleanup()
+  })
+
+  const renderPage = async () => {
+    const [wrapper] = await renderComponent(
+      editChannelContributorsURL(channel.name),
+      [
+        actions.subscribedChannels.get.requestType,
+        actions.subscribedChannels.get.successType,
+        actions.channels.get.requestType,
+        actions.channels.get.successType,
+        actions.channelContributors.get.requestType,
+        actions.channelContributors.get.successType,
+        actions.profiles.get.requestType,
+        actions.profiles.get.successType,
+        SET_CHANNEL_DATA
+      ]
+    )
+    return wrapper.update()
+  }
+
+  it("should set the document title", async () => {
+    await renderPage()
+    assert.equal(document.title, formatTitle("Edit Channel"))
+  })
```

Comment:
In this file and the moderators test file, I feel like you should at least check that the `Edit*Form` component is being rendered and that we're passing it a list of contributors/moderators